src/themes/OLH/templates/journal/editorial_team.html

```diff
@@ -17,23 +17,21 @@
             {% if group_id %}
                 <h1>{{ editorial_groups.0.name }}</h1>
             {% else %}
-                <h1>{% trans "Editorial Team" %}</h1>
+                <h1>{{ journal_settings.styling.editorial_group_page_name }}</h1>
             {% endif %}
             {% for group in editorial_groups %}
-                {% if not group_id %}
-                    <h4>{{ group.name }}</h4>
+                {% if not journal_settings.styling.hide_editorial_group_names %}
+                    {% if not group_id %}
+                        <h4>{{ group.name }}</h4>
```

Comment:
Is this a good opportunity to fix the heading structure and change this to an H2?